frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -475,6 +484,22 @@ function ReaderPaging:onZoomModeUpdate(new_mode)
     self.zoom_mode = new_mode
 end
 
+function ReaderPaging:onZoomPanUpdate(settings)
+    local _settings = {
+        "zoom_pan_h_overlap",
+        "zoom_pan_v_overlap",
+        "zoom_pan_bottom_to_top",
+        "zoom_pan_direction_vertical",
+    }
```

Comment:
Wouldn't it be better to change the scope of this variable, since it's used  in
```lua
    for _, v in ipairs({
        "zoom_pan_h_overlap",
        "zoom_pan_v_overlap",
        "zoom_pan_bottom_to_top",
        "zoom_pan_direction_vertical",
    }) do```
This way we could avoid duplicating this.